backend/src/game/game.service.ts

```diff
@@ -77,17 +347,34 @@ export class GameService {
     }
   }
 
-  async createWaitingGame(player: Player){
-    const gameID = player.socketID;
+  /****************************************************************************/
+  /* GAME INIT                                                                */
+  /****************************************************************************/
+
+  setCanvas({
+    canvasHeight,
+    paddleHeight,
+  }: {
+    canvasHeight: number;
+    paddleHeight: number;
+  }) {
+    this.canvas.canvasHeight = canvasHeight;
+    this.canvas.paddleHeight = paddleHeight;
+    this.startPaddle = canvasHeight / 2 - paddleHeight / 2;
+  }
+
+
+  async createWaitingGame(player: Player) : Promise<Game> {
```

Comment:
Small detail, but I would place this function under 'game create' in stead of 'game init'